docs/CompatibleSoftware.rst

```diff
@@ -12,171 +12,354 @@ The following software all supports OpenColorIO (to varying degrees).
+
+- `Managing Looks with OpenColorIO <https://www.isotropix.com/learn/tutorials/managing-looks-with-opencolorio-ocio>`__
+
+
+CryEngine (CryTek)
+*******************
+
+CryEngine is a real-time game engine, targeting applications in the motion-picture market. While we don't know many details about the CryEngine OpenColorIO integration, we're looking forward to learning more as information becomes available.
+
+Website: `<https://www.cryengine.com/>`__
+
+
+DJV
+***
+
+Website: `<https://darbyjohnston.github.io/DJV/>`__
+
+
+Gaffer
+******
+
+Gaffer is a node based application for use in CG and VFX production, with a particular focus on lighting and look development.
+
+Website: `<http://www.gafferhq.org/>`__
+
+
+Hiero (Foundry)
+***************
+
+Hiero ships with native support for OCIO in the display and the equivalent of Nuke's OCIOColorSpace in the Read nodes. It comes with the "nuke-default" OCIO config by default, so the Hiero viewer
+matches when sending files to Nuke for rendering.
+
+Website: `<https://www.foundry.com/products/hiero>`__
+
+Supported version: >= 1.0
+
+
+Houdini (SideFX)
+****************
+
+Website: `<https://www.sidefx.com>`__
+
+Supported Version: >= 16
+
+Documentation:
+
+
+- `OpenColorIO Support <https://www.sidefx.com/docs/houdini/io/ocio.html>`__
+
+- `What's new in Houdini 16 <https://www.sidefx.com/filmtv/whats-new-h16/>`__
+
+
+Java (Beta)
+***********
+
+The OpenColorIO API is available for use in Java. This integration is currently considered a work in progress, and should not be relied upon for critical production work.
+
+Website: `<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/bindings/java>`__
+
+
+Katana (Foundry)
+****************
+
+Website: `<http://www.thefoundry.co.uk/products/katana>`__
+
+Documentation:
+
+- `Managing Color <https://learn.foundry.com/katana/Content/ug/rendering_scene/managing_color.html>`__
+
+- `Color Nodes <https://learn.foundry.com/katana/Content/rg/color_2d.html>`__
 
 
 Krita
 *****
 
-`2D Paint - Open Source <http://www.krita.org/>`__
+Krita now support OpenColorIO for image viewing, allowing for the accurate painting of float32/OpenEXR imagery.
+
+Website: `<http://www.krita.org/>`__
+
+Documentation :
+
+- `Krita Starts Supporting OpenColorIO <https://krita.org/en/item/krita-starts-supporting-opencolorio/>`__.
+
+
+Mari (Foundry)
+**************
+
+Website: `<https://www.foundry.com/products/mari>`__
+
+Supported Version: >= 1.4v1
+
+Documentation:
+
+- `Managing Colors in Mari <https://learn.foundry.com/mari/4.0/Content/user_guide/managing_colors/anaging_colors_in_mari.html>`__
+
+- `Color Management <https://learn.foundry.com/mari/4.0/Content/user_guide/managing_colors/color_management.html>`__
 
-Krita now support OpenColorIO for image viewing, allowing for the accurate
-painting of float32/OpenEXR imagery.
 
-See `krita.org 
-<http://www.krita.org/item/113-krita-starts-supporting-opencolorio>`__
-for details.
+Maya (Autodesk)
```

Comment:
This section has invalid formatting in a reST viewer, and one of the doc links was removed. Should be:

```
Maya (Autodesk)
***************

Autodesk Maya is a 3D computer animation, modeling, simulation, and rendering software.

Website: `<https://www.autodesk.com/products/maya/overview>`__

Supported Version: >= 2016

Documentation:

- `Use an OCIO configuration file for color management <https://knowledge.autodesk.com/support/maya-lt/learn-explore/caas/CloudHelp/cloudhelp/2016/ENU/MayaLT/files/GUID-4AE9AABC-A900-4276-AFE4-F9DCD96BA03D-htm.html>`__
- `What's New in Rendering <https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2016/ENU/Maya/files/GUID-C22F815A-8390-405B-BA50-74FEC42C75E0-htm.html>`__

```